packages/file/lib/src/interface/file_system.dart

```diff
@@ -52,23 +53,23 @@ abstract class FileSystem {
 
   /// Sets the current working directory to the specified [path].
   ///
-  /// The new value set can be either a [Directory] or a [String].
+  /// The value set can be either a [Directory] or a [String].
```

Comment:
Here and the one above, do we still want new in the comment?

Reply:
Ooh, `sed` gone wild.  I'll fix.